octoprint_mrbeam/static/js/app/view-models/wizard/netconnectd-wrapper.js

```diff
@@ -37,10 +37,7 @@ $(function () {
             });
         };
 
-        self.onConnectionOptionSelection = function (
-            triggerElement,
-            divElementToShow
-        ) {
+        self.showSelectedOption = function (triggerElement, divElementToShow) {
```

Comment:
Is it not more like add Click event for connection options?

Reply:
Not necessary